### ISSUE TITLE
VS Integration: wrong hint for variables

### ISSUE THREAD

**Issue by leon-ts** [X-Sharp/XSharpPublic#1260]
Variables are displayed incorrectly in lines starting from the second line.

**Example:**
![Screenshot4](https://github.com/X-Sharp/XSharpPublic/assets/64518775/87a67682-b5a5-4e29-927d-8d2e682146eb)

**Environment:**
X# 2.16.0.5
IDE VS2019

**Comment by cpyrgas** [comment]
Robert, has this fix also covered the following, or should I open a separate report?

LOCAL n AS INT
n := 1 + ;
System. // no member completion here
// and neither that gives member completion in the continued line: 
// n:

**Comment by RobertvanderHulst** [comment]
Yes, that is fixed now as well.

**Comment by leon-ts** [comment]
Robert,
The problem I described in this ticket has been fixed. But the problem that Chris described remains: after entering a namespace such as System, there are no prompts. At the same time, there are hints for static classes, as well as for instances (like 'n' in Chris' example). But as I found out, the hints for System and other namespaces is no longer displayed in all cases, even where there is no line separator. Therefore, I am closing this ticket as fixed and will create a new ticket with a different problem.
